sim/pokemon.ts

```diff
@@ -671,6 +671,7 @@ export class Pokemon {
 	getAtLoc(targetLoc: number) {
 		let side = this.battle.sides[targetLoc < 0 ? this.side.n % 2 : (this.side.n + 1) % 2];
 		targetLoc = Math.abs(targetLoc);
+		if (this.battle.gameType === 'rotation') return side.pokemon[targetLoc - 1];
```

Comment:
Why is this needed, in the sense that why would anyone be trying to target position 2 or 3 during a rotation battle?

Reply:
When rotating and using a self-targetting move targetLoc is set as -2 or -3, which is beyond the bounds of side.active. This causes the `if` below to assume it's a multi battle and index a side that doesn't exist, which crashes.

Reply:
Ah, so could my suggestion of a side property/method for "active except first 3 if rotation battle" take care of this case too?